android/app/build.gradle

```diff
@@ -47,7 +50,7 @@ android {
         applicationId "com.example.movies_and_stars"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 21
```

Comment:
```suggestion
        minSdkVersion flutter.minSdkVersion
```

Reply:
<img width="837" alt="Screenshot 2022-10-12 at 19 18 12" src="https://user-images.githubusercontent.com/111364147/195395715-c38adc20-44c7-4c8b-8581-21e863222d12.png">

Reply:
search where flutter.minSdkVersion is set and change it here